docs/variant-specific/pink.mdx

```diff
@@ -291,37 +291,33 @@ These conventions apply to any variant with a pink (touched by all ranks) suit.
 - _Bubblegum Bluffs_ can only be performed on non-pink cards; otherwise, it will look like a _Bubblegum Finesse_.
 - Note that it is impossible to _3 Bluff_ and _Bubblegum Bluff_ at the same time.
 
-### The Bubblegum Bluff, part 2
+### The Bubblegum Bluff (Two-Away-From-Playable Form)
 
-- Note that, as with a standard _Bluff_, the receiver of a _[Bubblegum Bluff](#the-bubblegum-bluff)_ marks their pink card as the lowest-possible logical _Good Touch_ value it could be.
-- However, it is sometimes possible that the lowest-possible _Good Touch_ value is actually _two-away-from-playable_.
+- Normally, _Bubblegum Bluffs_ are always performed by touching _one-way-from-playable_ pink cards.
+- Rarely, it is possible to perform a _Bubblegum Bluff_ on a _two-away-from-playable_ pink card. However, this can only be done in a specific situation where, due to the circumstances of the clue, it would be impossible for the clued card to be _one-way-from-playable_.
```

Comment:
@Zamiell obviously, you thought my proposed wording was unclear, hence the changes 😛 

but I sort of disagree with this wording (which reminds me more of the pre-PR document).
it's not really a "specific situation" or "circumstance" -- it's precisely the _logic_ following from _Good Touch_ and lowest-value.

what were your thoughts on this?

Reply:
does it have to do with "good touch" though?
like, say for example you do a "2 save" and it causes a crazy blind play, then it can't be a 2 save anymore, and it is a taffy bluff as a 3.
in that example, the operant principle is the fact that the "2 save" convention already exists and that since it exists, it wouldn't cause any blind plays.
but the doc can't be worded like this:
> Sometimes, a clue would look like a _2 Save_, but it can't be a _2 Save_, so then it would have to be on the next _two-away-from-playable_ card.
so we need to word that sentence in a more generic way, so i chose the phase "circumstances of the clue", but you can workshop this if you don't like it.

Reply:
ah okay I see what you're saying.

the "logic" relies on other conventions as well, like 2 saves existing.

but in a way, that's the point. this convention is built on top of those existing ones. we know that a 2 save doesn't cause blind plays. so if there is a blind play, the card can't be a 2.

and we agree that lying about a card's identity needs to be fixed in a certain way (blind play from number slot).

given those facts, the card being one away or two away isn't really important -- it's the lowest possible value it could be, assuming the card is useful ("good touch").

That's precisely where (lowest possible and) good touch comes in: "what value is the card?"

Reply:
I updated the doc a bit if you want to review the pr again and suggest more edits (maybe we should use the GitHub suggest edit feature, if you're familiar with that)

Reply:
sorry been traveling, on mobile right now, I can take a look and try to suggest something